README.md

```diff
@@ -105,4 +105,4 @@ Third Party Packages
 - https://github.com/ramsey/uuid MIT Licence https://github.com/ramsey/uuid/blob/master/LICENSE
 - https://github.com/sebdesign/laravel-state-machine MIT Licence https://github.com/sebdesign/laravel-state-machine/blob/master/LICENSE.md
 - https://github.com/spinen/laravel-mail-assertions MIT Licence https://opensource.org/licenses/MIT
-- https://github.com/TomLingham/Laravel-Searchy MIT Licence https://github.com/TomLingham/Laravel-Searchy/blob/master/LICENSE
```

Comment:
I have no idea what changed here, I used the github inline editor for the above change.